scripts/ci/pin-to-cluster.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+echo "::group::pin add"
+ipfs-cluster-ctl --enc=json \
+    --host "/dnsaddr/ipfs-websites.collab.ipfscluster.io" \
+    --basic-auth "${CLUSTER_USER}:${CLUSTER_PASSWORD}" \
+    pin add \
+    --pin-name="${PIN_NAME}" \
+    --no-status $PIN_ADD_EXTRA_ARGS \
+    "$PIN_CID"
+echo "::endgroup::"
+
+echo "::group::waiting until pinned"
+    while true; do
+    ipfs-cluster-ctl --enc=json \
+        --host "/dnsaddr/ipfs-websites.collab.ipfscluster.io" \
+        --basic-auth "${CLUSTER_USER}:${CLUSTER_PASSWORD}" \
+        status "$PIN_CID" | tee cluster-pin-status
+    if [[ $(jq '.peer_map[].status' cluster-pin-status | grep '"pinned"' | wc -l) -ge 2 ]]; then
+        echo "Got 2 pin confirmations, finishing the workflow"
```

Comment:
:information_source: to make CI faster and avoid hanging forever if parts of cluster are out of sync, cluster pinning is considered "done" after we have 2 copies. 

Here, I do it manually in userland, but filled https://github.com/ipfs/ipfs-cluster/issues/1427 to add this sort of thing to ipfs-cluster-ctl CLI tool.

Reply:
Similar to above: when do we decide to put this in the PR vs. the code itself?  Do folks tend to do a git blame and look for comments to find more context?

Reply:
I often use git blame,  but in this specific case those comments are provided mostly to make PR review easier.

Reply:
Ok.  I'll defer to the team for the standard.  I was bringing this up because it was hard for me to imagine a case where a comment is useful during PR where it isn't also useful 6 months in the future when looking at the code with fresh eyes.  Anyways, sounds good.